src/types/state.js

```diff
@@ -281,6 +281,7 @@ export type FullProfileSpecificUrlState = {|
   hiddenGlobalTracks: Set<TrackIndex>,
   hiddenLocalTracksByPid: Map<Pid, Set<TrackIndex>>,
   localTrackOrderByPid: Map<Pid, TrackIndex[]>,
+  localTrackOrderChangedByPid: Set<Pid>,
```

Comment:
I was thinking for a while whether there might be a better name for this. The current name sounds like a `Map<Pid, bool>`, and it's also not clear that we only track changes initiated by the user and not by us.

But `pidsWhoseLocalTrackOrderHasEverBeenManipulatedByTheUser` just doesn't roll off the tongue that easily.
Maybe you can add a comment to the field, or to the reducer.

Reply:
> I was thinking for a while whether there might be a better name for this. The current name sounds like a `Map<Pid, bool>`, and it's also not clear that we only track changes initiated by the user and not by us.

Funny thing is that I used a `Map<Pid, bool>` initially 😅

> 
> But `pidsWhoseLocalTrackOrderHasEverBeenManipulatedByTheUser` just doesn't roll off the tongue that easily. Maybe you can add a comment to the field, or to the reducer.

I agree with you that the name could be better, I'll think about that.

Reply:
What about `localTrackOrderChangedPids`?